### PR TITLE
MM-25000: In-Product Discovery

### PR DESCRIPTION
**Pull request by mkraft**
#### Summary

Adds admin console menu items linking to HTML views advertising the enterprise features.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-25000

#### Screenshots

Screen Shot 2020-05-13 at 8 29 21 PM
Screen Shot 2020-05-13 at 8 29 26 PM

### CODE REVIEW

File: components/admin_console/feature_discovery/feature_discovery.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.FeatureDiscovery_copy {
+    font-size: 14px;
+    line-height: 20px;
+    padding-bottom: 24px;
+}
+
+.FeatureDiscovery .btn {
+    margin-right: 24px;
+    width: 160px;
```
Comment (mkraft) [inline components/admin_console/feature_discovery/feature_discovery.scss:45]: Button width decrease requested by UX.

File: components/invitation_modal/invitation_modal_members_step.scss [hunk 2]
```diff
@@ -108,23 +108,32 @@
...
+            background: var(--center-channel-color-08);
+            color: var(--center-channel-color-56);
+        }
+        &.btn-secondary {
+            background: var(--center-channel-bg);
+            border: 1px solid var(--button-bg);
+            color: var(--button-bg);
+        }
+    }
+}
```
Comment (mkraft) [inline components/invitation_modal/invitation_modal_members_step.scss:139]: This was the [recommended approach](https://community-daily.mattermost.com/core/pl/b1ow1n9ee3d79cw5tex33dujno) by the apps team pending the common components.

**Comment by mkraft** [comment]
@thefactremains Could you please use the screenshots to review the copy. Also, I require the primary and secondary call-to-action URLs (with any desired tracking information).

@michaelgamble Could you please use the test server to review the pixel-perfectness, hovers, responsiveness, etc.?

File: components/admin_console/feature_discovery/feature_discovery.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+.FeatureDiscovery {
+    background: var(--center-channel-bg);
+    box-shadow: 0px 2px 3px rgba(0, 0, 0, 0.08);
+    border-radius: 8px;
+    padding: 20px;
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: normal;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(440px, 1fr));
```
Comment (mkraft) [inline components/admin_console/feature_discovery/feature_discovery.scss:12]: Thanks @michaelgamble for tipping me off to the `grid-template-columns` property.

**Comment by michaelgamble** [comment]
it may be just a matter of adjusting the text line-height sometimes setting it to the same as the font-size and then increasing the parent objects padding in order to keep the overall height 48px

**Comment by mkraft** [comment]
> it may be just a matter of adjusting the text line-height sometimes setting it to the same as the font-size and then increasing the parent objects padding in order to keep the overall height 48px

@michaelgamble Can you please confirm that the change you're requesting applies to the invitation flow too?

**Comment by michaelgamble** [comment]
> > it may be just a matter of adjusting the text line-height sometimes setting it to the same as the font-size and then increasing the parent objects padding in order to keep the overall height 48px
> 
> @michaelgamble Can you please confirm that the change you're requesting applies to the invitation flow too?

Yep, not sure when that happened, but the issue exists their too

**Comment by mkraft** [comment]
> > > it may be just a matter of adjusting the text line-height sometimes setting it to the same as the font-size and then increasing the parent objects padding in order to keep the overall height 48px
> > 
> > 
> > @michaelgamble Can you please confirm that the change you're requesting applies to the invitation flow too?
> 
> Yep, not sure when that happened, but the issue exists their too

Requested changes are in.

**Pull request merged**